### PR TITLE
Feature flags implementation

### LINKED ISSUE #6: Enable feature flags, colorscheme, and avatarURL
Need research and testing with these fields that the SDK takes in. Research for feature flags and color scheme because there is not great documentation on Jitsi site on what to pass in. AvatarURL was tried, but did nothing on the UI (like email address).

### LINKED ISSUE #15: can not deploy app in app store
Apple reply.
This app cannot be approved with CallKit functionality active in China. Please make the appropriate changes and resubmit this app for review.

If you have already ensured that CallKit functionality is not active in China, you may reply to this message in Resolution Center to confirm.

Voice over Internet Protocol (VoIP) call functionality continues to be allowed but can no longer take advantage of CallKit’s intuitive look and feel. CallKit can continue to be used in apps outside of China.

Please help.

### LINKED ISSUE #21: How to remove chat & other options?
Hi I'm using flutter_jitsi_meet,
I need some help in setting up.
How to remove chat and other options in more options?
Please help me ASAP

@gunschu thanks in Advance.

### LINKED ISSUE #22: Is there a way to fully customize the conference widget?
I would like to know if there is a way to fully customize the conference widget?

Congratulations for your job!

### LINKED ISSUE #26: How can i remove minimize icon button inside jitsi_meet screen
How can I remove the minimize icon button inside the jitsi_meet screen?
If anybody knows please share your knowledge, it's useful for me :)

### PR DESCRIPTION
**Pull request by TBG-FR**
Improved implementation of feature flags. 
**TLDR : All feature flags are now available through the plugin.**

#### Modifications

- [X] Created a list of flags based on [Jitsi Meet constants](https://github.com/jitsi/jitsi-meet/blob/master/react/features/base/flags/constants.js), as an `Enum`
- [X] Added flags descriptions (in comments) and values (in a `Map`)
- [X] Removed `.setWelcomePageEnabled(false)` from Kotlin code as it uses feature flags under the hood ([see here](https://github.com/jitsi/jitsi-meet/blob/master/android/sdk/src/main/java/org/jitsi/meet/sdk/JitsiMeetConferenceOptions.java#L234))
- [X] Modified Android/Kotlin code to get and use the `Map` of feature flags instead of "static" fields
- [X] Modified iOS/Swift code to get and use the `Map` of feature flags instead of "static" fields
- [X] Modified `example` app to show how to use flags
- [X] Updated README with informations about feature flags

#### Why ?

- We now have **only an `Enum` and a `Map` to modify if new feature flags are added** in Jitsi Meet SDK, we don't need to add attributes/fields and everything
- Meeting options and feature flags are a bit more separated (no longer need one field in `JitsiMeetOptions` for each flag)
- It's **more convenient to use** (developers just fill a map with flags they need to modify)
- It's safer to use (thanks to the `Enum` and `Map`)
- If **no flags are provided, default values from the SDK will be used** (we no longer need to put something with `flagname ? : false` and set an arbitrary `true` or `false` default setting

#### Reviews

I tested that modification under virtual devices, both on Android and iOS. If you want to test it yourself, go on, I'd be glad to get some feedback, and even maybe improvements 😉

### CODE REVIEW

**Comment by rwbr** [comment]
I reviewed your changes and in my opinion everything looks fine. A test in the iOS-Simulator delivered a positive result also. I think, this PR should be accepted.

**Comment by TBG-FR** [comment]
Thanks a lot for taking the time to review and test that, and your feeback 😃
